cascade/auditor.py

```diff
@@ -0,0 +1,80 @@
+import numpy as np
+from scipy.stats import multivariate_normal
+from ase import Atoms
+
+class BaseAuditor: 
```

Comment:
Shall we document what this is and how to use it before we get too much farther?